src/schema/electronic-travel-authorisation.json

```diff
@@ -41,6 +41,18 @@
         "application": {
             "typeRef": "#/domains/ElectronicTravelAuthorisation/types/Application"
         },
+        "passportBioDataPagePhoto": {
+            "typeRef": "#/domains/Generic/types/StructuredTextItems",
+            "description": "A photo with passport details section. Used for passport validation."
+        },
+        "otherDocuments": {
```

Comment:
How do we know what each item under "otherDocuments" is? Say you have an array of 3 items: "entry permit of the host", "identity card", "business visit invitation". How do you know which is which?

Instead of "otherDocuments" with an unknown list of documents, please consider specific documents under "applicant".

As far as passport photos go, you could add them under Passport (there's an existing data type).

Reply:
You can add multiple documents under other documents section, so order doesn't matter much. But we can reduce it to one, lets say "Identity card"? 
I will move the passport photos under the passport type.

Reply:
if you want to have a single additional scan, then call it what it is, e.g. "identityCardPhoto", not "otherDocuments".

Reply:
I want to have multiple, like it is now. But yes, will rename the key if multiple are not necessary and we choose just one document.